ui/src/main/java/edu/wpi/grip/ui/Main.java

```diff
@@ -25,6 +28,35 @@ public static void main(String[] args) {
 
     @Override
     public void start(Stage stage) {
+
+        //Set up the global level logger. This handles IO for all loggers.
+        Logger globalLogger = LogManager.getLogManager().getLogger("");//This is our global logger
+
+        //Remove the default handler(s) so we can add our own
+        Handler[] handlers = globalLogger.getHandlers();
+        for(Handler handler : handlers) {
+            globalLogger.removeHandler(handler);
+        }
+
+        Handler fileHandler  = null;//This will be our handler for the global logger
+
+        try{
+            fileHandler  = new FileHandler("./GRIPlogger.log");//Log to the file "GRIPlogger.log"
+
+            globalLogger.addHandler(fileHandler);//Add the handler to the global logger
```

Comment:
If the "main" method is run then we are running with gradle. Otherwise "start" is the first method called by javafx.